### PR TITLE
Changes after user test

### PR DESCRIPTION
**Pull request by tschubotz**

### CODE REVIEW

File: docs/sdks_03_safe_apps.md [hunk 1]
```diff
@@ -4,23 +4,38 @@ title: Safe Apps
...
 ---
 
+## Introduction
+
 With over $1B worth of digital assets held in Gnosis Multisigs alone, it’s essential for the dapp ecosystem to access the untapped market of multi-signature wallets. Safe Apps introduce a completely new way for developers to build their dapps right into a Multisig interface. Turn your Dapp into an HTML iframe component that can be accessed through the Safe Multisig. We are providing extensive developer tooling to make it easy to develop, test and integrate Safe Apps into the Safe Multisig. This includes a unique design system, reusable components and a Safe App SDK that facilitates the Safe App <> Safe Multisig communication.
 
 Please refer to this [EthCC3 talk](https://www.youtube.com/watch?v=1GirpNHZPJM) to learn more about Safe Apps.
 
-## Safe Apps SDK
+There are already Safe apps available for a number of popular protocols such as [Aave](https://aave.com/), [Synthetix](https://synthetix.io/), [1inch](https://1inch.exchange/) or [Balancer](https://balancer.finance/). These have been build by 3rd party developers or the projects themselves.
```
Comment (rmeissner) [inline docs/sdks_03_safe_apps.md:13]: Capitalize a in Apps

File: docs/sdks_03_safe_apps.md [hunk 2]
```diff
@@ -29,6 +44,22 @@ We provide Safe App developers with reusable react components to make it easy to
 [UI Guidelines](assets/safe_apps_guides.png)
 
 
+### Existing Safe apps
```
Comment (rmeissner) [inline docs/sdks_03_safe_apps.md:47]: We need to agree on capitalized A or not :D cc @lukasschor

File: docs/sdks_03_safe_apps.md [hunk 3]
```diff
@@ -29,6 +44,22 @@ We provide Safe App developers with reusable react components to make it easy to
...
+- [Open Zeppelin](https://github.com/OpenZeppelin/upgrades-safe-app)
+- [Balancer - Pooling](https://github.com/TomAFrench/pool-management/tree/safe-app)
+- [Balancer - Exchange](https://github.com/balancer-labs/balancer-exchange)
+- [1inch Exchange](https://github.com/CryptoManiacsZone/gnosis.1inch.exchange)
+- [Synthetix](https://github.com/protofire/safe-app-synthetix-mintr)
+- [Sablier](https://github.com/TomAFrench/sablier-safe-app)
+- [Compound](https://github.com/gnosis/safe-react-apps/tree/development/src/apps/Compound)
+- [Transaction builder](https://github.com/gnosis/safe-react-apps/tree/development/src/apps/TxBuilder)
+- [Aave](https://github.com/mrdavey/gnosis-safe-app)
+- [Idle](https://github.com/krzysu/safe-app-idle)
```
Comment (rmeissner) [inline docs/sdks_03_safe_apps.md:60]: Should we link the list from the other pr?

File: docs/sdks_03_safe_apps.md (hunk 2 above)
Comment (tschubotz) [reply docs/sdks_03_safe_apps.md:47]: capital A sounds good, will change it.

File: docs/sdks_03_safe_apps.md (hunk 3 above)
Comment (tschubotz) [reply docs/sdks_03_safe_apps.md:60]: Here we link the code whereas in the other lists only the projects, so would keep separate.

**Pull request merged**